### ISSUE TITLE
How can I show uibackground call? 

### ISSUE THREAD

**Issue by zettry1** [flutter-webrtc/callkeep#138]
Hello, Is there a way to show like this in callkeep?. Do I need more configuration or something?

![](https://i.stack.imgur.com/9XogR.png)

**Comment by efraespada** [comment]
No, I suppose it is a native system element and tells the user some hardware (camera and mic) are being used.
